data/soapvalidator/Tasks/Get-Tasks.xml

```diff
@@ -141,8 +141,8 @@
         <t:response>
             <t:select path = "//mail:GetTaskResponse/mail:task/mail:inv/mail:comp" attr="name" match="${task1.subject}"/>
             <t:select path = "//mail:GetTaskResponse/mail:task/mail:inv/mail:comp" attr="loc" match="${task1.location}"/>
-            <t:select path = "//mail:GetTaskResponse/mail:task/mail:inv/mail:comp/mail:s" attr="d" match="${Time1}"/>
-            <t:select path = "//mail:GetTaskResponse/mail:task/mail:inv/mail:comp/mail:e" attr="d" match="${Time2}"/>
+            <t:select path = "//mail:GetTaskResponse/mail:task/mail:inv/mail:comp/mail:s" attr="d" contains="${Time1}"/>
```

Comment:
Do you know why Z was added in start and end time? This testcase is not creating task with timezone.

Reply:
Z is shorthand for the UTC timezone.

Reply:
In above case, timezone info was not passed in create appointment/task request. We should keep match validation and validate for Z.

Reply:
Fixed test cases to add match validation and validation of Z